testdata-seg-fault/in/tables/in.c-ex-dummy.dummy

```diff
@@ -0,0 +1,2 @@
+"col1","col2"
+"abc","0020-02-18"
```

Comment:
Z nejakeho dovodu TDE knicnica nevie parsnut datum `0020-02-18`. 
Je to 6 rokov stara binarka, ... mame task na novu verziu, ... ale do logov chodi vela app. errors, tak to musime nejak fixnut aj tu.